### PR TITLE
#137 update checkbox icon

### LINKED ISSUE #137: Update checkbox icon
Update the icon used by the checkbox component to more closely match the design in zeplin

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #137 
and then gets a bit carried away with icons in storybook. What I was trying to do, was help people developing later, so that they can find the right icon easily.

### CODE REVIEW

File: packages/common/src/ui/icons/CheckboxDisabled.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export const CheckboxDisabled = (props: SvgIconProps): JSX.Element => {
```
Comment (joshxg) [inline packages/common/src/ui/icons/CheckboxDisabled.tsx:4]: Maybe `CheckboxIndeterminate`? 🤔  An actual disabled checkbox I think just being a light grey of either checked or unchecked?

File: packages/common/src/ui/icons/Icon.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  | 'success'
+  | 'warning';
+
+interface SvgIconProps {
+  color: Color;
+}
+export const Primary = Template.bind({});
+export const Secondary = Template.bind({});
+
+Secondary.args = { color: 'secondary' } as SvgIconProps;
```
Comment (joshxg) [inline packages/common/src/ui/icons/Icon.stories.tsx:131]: This is really awesome and well worth the extra effort

**Review by joshxg — APPROVED** [review]
👍  awesome, thanks for the svgs!

File: packages/common/src/ui/icons/CheckboxDisabled.tsx (hunk 1 above)
Comment (mark-prins) [reply packages/common/src/ui/icons/CheckboxDisabled.tsx:4]: Yep. good idea - I took the name from the svg I stole; but Indeterminate is more appropriate.

**Pull request merged**